spec/requests/api/v1/driver_commission_histories_spec.rb

```diff
@@ -1,24 +1,64 @@
 require 'rails_helper'
 
 RSpec.describe '/api/v1/all', :type => :request do
```

Comment:
I'm in favor of it being named

```ruby
RSpec.describe '/api/v1/driver_commissions_history', :type => :request do
```

And this the driver request specs would be named:

```ruby
RSpec.describe '/api/v1/drivers', :type => :request do
```